src/bin/worker.js

```diff
@@ -92,6 +92,10 @@ program.parse(process.argv);
 
 // Main.
 (async () => {
+  process.on('unhandledRejection', (reason, p) => {
```

Comment:
I think that lib-monitor catches this and sends to sentry.  Is this for just more verbose logging of such things?

Reply:
When running tests locally, I ran into some "unhandled reject" errors and had to add this code to figure them out.